### PR TITLE
BH-270: Upgrade ES 7.10 and Mysql to 8.0.22

### PR DESCRIPTION
**Pull request by mr-prud**
**Description (for Contributor and Core Developer)**

Upgrade Mysql and ES to have supported stack  as long as the AKENEO PIM V5

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by jmleroux — APPROVED** [review]

File: src/Akeneo/Pim/Enrichment/Bundle/Elasticsearch/Filter/Field/AncestorIdFilter.php [hunk 1]
```diff
@@ -71,11 +71,9 @@ public function addFieldFilter($field, $operator, $values, $locale = null, $chan
             case Operators::IN_LIST:
                 $this->searchQueryBuilder->addShould(
                     [
-                        [
```
Comment (jmleroux) [inline src/Akeneo/Pim/Enrichment/Bundle/Elasticsearch/Filter/Field/AncestorIdFilter.php:74]: Is it related to ES 7.10 or was it a bug?

**Review by jmleroux — APPROVED** [review]
GTM with a question

File: src/Akeneo/Pim/Enrichment/Bundle/Elasticsearch/Filter/Field/AncestorIdFilter.php (hunk 1 above)
Comment (mr-prud) [reply src/Akeneo/Pim/Enrichment/Bundle/Elasticsearch/Filter/Field/AncestorIdFilter.php:74]: we need to ask @adrienwb for that

Comment (mmetayer) [reply src/Akeneo/Pim/Enrichment/Bundle/Elasticsearch/Filter/Field/AncestorIdFilter.php:74]: it's related to ES 7.10

**Review by mmetayer — APPROVED** [review]

**Pull request merged**